web/src/model/flow-query.ts

```diff
@@ -8,7 +8,7 @@ export type PacketLoss = 'dropped' | 'hasDrops' | 'sent' | 'all';
 export type MetricFunction = 'count' | 'sum' | 'avg' | 'min' | 'max' | 'p90' | 'p99' | 'rate';
 export type StatFunction = MetricFunction | 'last';
 export type MetricType = 'Flows' | 'DnsFlows' | Field;
-export type FlowScope = 'app' | 'cluster' | 'zone' | 'host' | 'namespace' | 'owner' | 'resource';
+export type FlowScope = 'app' | 'cluster' | 'zone' | 'host' | 'namespace' | 'owner' | 'resource' | string;
```

Comment:
since it's now config based maybe just `FlowScope = string` is enough? (or is there still any good benefit from typing some of known scopes?)
same remark for groups below.

Reply:
there are some specific cases indeed especially for `owner` & `resource` but I will remove the ones not used

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/634/commits/d5b39583752b42e1247e03b547af165ee6bac427